.github/workflows/ci-bun.yml

```
@@ -0,0 +1,36 @@
+name: CI Bun
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # Run on every PR, regardless of branch
+    branches: [ '*' ]
+
+jobs:
+  test:
+    name: Bun CI
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        bun-version: [1.0.0, 1.1.0, 1.2.0, 1.3.0]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: ${{ matrix.bun-version }}
+
+      # TODO: Remove this step once the lockfile has been updated to v3 in the repository
+      - name: Install npm dependencies (lockfile v3)
+        run: npm i --lockfile-version 3 --frozen-lockfile
```

huh, `--frozen-lockfile` isn't a documented npm cli flag

I know this is from the `bun install` error message but Im wondering if they made a mistake, this is a yarn flag for sure though

for npm we'd have to `npm ci` or even better we can _only_ update the lockfile without installing anything via `--package-lock-only`
```suggestion
        run: npm i --lockfile-version 3 --package-lock-only
```